raft/minter.go

```diff
@@ -89,7 +89,7 @@ func newMinter(config *params.ChainConfig, eth *RaftService, blockTime time.Dura
 		speculativeChain: newSpeculativeChain(),
 
 		invalidRaftOrderingChan: make(chan InvalidRaftOrdering, 1),
-		chainHeadChan:           make(chan core.ChainHeadEvent, 1),
+		chainHeadChan:           make(chan core.ChainHeadEvent, 10),
```

Comment:
Instead of hardcoded value can we use `tx_pool.go::chainHeadChanSize` constant, in case if the buffer size is changed later, it is changed at all the places

Reply:
Changed to use the chainHeadChanSize constant instead of hard coded value